pkg/trace/api/api_test.go

```diff
@@ -615,46 +592,36 @@ func TestClientComputedStatsHeader(t *testing.T) {
 	mux := rcv.buildMux()
 	server := httptest.NewServer(mux)
 
-	// run runs the test with ClientComputedStats turned on.
-	run := func(on bool) func(t *testing.T) {
-		return func(t *testing.T) {
-			bts, err := testutil.GetTestTraces(10, 10, true).MarshalMsg(nil)
-			assert.Nil(t, err)
+	bts, err := testutil.GetTestTraces(10, 10, true).MarshalMsg(nil)
+	assert.Nil(t, err)
 
-			req, _ := http.NewRequest("POST", server.URL+"/v0.4/traces", bytes.NewReader(bts))
-			req.Header.Set("Content-Type", "application/msgpack")
-			req.Header.Set(headerLang, "lang1")
-			if on {
-				req.Header.Set(headerComputedStats, "yes")
-			}
-			var wg sync.WaitGroup
-			wg.Add(1)
-			go func() {
-				defer wg.Done()
-				resp, err := http.DefaultClient.Do(req)
-				if err != nil {
-					t.Fatal(err)
-				}
-				if resp.StatusCode != 200 {
-					t.Fatal(resp.StatusCode)
-				}
-			}()
-			timeout := time.After(time.Second)
-			for {
-				select {
-				case p := <-rcv.out:
-					assert.Equal(t, p.ClientComputedStats, on)
-					wg.Wait()
-					return
-				case <-timeout:
-					t.Fatal("no output")
-				}
-			}
+	req, _ := http.NewRequest("POST", server.URL+"/v0.4/traces", bytes.NewReader(bts))
+	req.Header.Set("Content-Type", "application/msgpack")
+	req.Header.Set(headerLang, "lang1")
+	req.Header.Set(headerComputedStats, "yes")
+
+	var wg sync.WaitGroup
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		resp, err := http.DefaultClient.Do(req)
+		if err != nil {
+			t.Fatal(err)
+		}
+		if resp.StatusCode != 200 {
+			t.Fatal(resp.StatusCode)
+		}
+	}()
+	timeout := time.After(time.Second)
+	for {
+		select {
+		case <-rcv.out:
+			wg.Wait()
+			return
+		case <-timeout:
+			t.Fatal("no output")
 		}
 	}
-
-	t.Run("on", run(true))
-	t.Run("off", run(false))
```

Comment:
Why did you remove the test for when the clientComputedStats header is not set?

Reply:
this is a mistake, changed the wrong test